nexus-config/src/nexus_config.rs

```diff
@@ -827,6 +826,20 @@ impl Default for SpEreportIngesterConfig {
     }
 }
 
+#[serde_as]
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+pub struct ChickenSwitchesConfig {
+    /// period (in seconds) for periodic activations of this background task
+    #[serde_as(as = "DurationSeconds<u64>")]
+    pub period_secs: Duration,
+}
+
+impl Default for ChickenSwitchesConfig {
+    fn default() -> Self {
+        Self { period_secs: Duration::from_secs(5) }
```

Comment:
Does this need a default? (If yes, why is it 5 here but 30 in the config files?)